src/device.ts

```diff
@@ -34,3 +34,13 @@ export const dismiss = (onCompletionUrl?: string): Promise<void> =>
             onCompletionUrl,
         },
     });
+
+type VibrationsAvailable = 'error' | 'success';
```

Comment:
`type VibrationType = 'error' | 'success';`

but you could simply define that inline

Reply:
oops, true